Task_3/Human.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_3
+{
+    abstract class Human
+    {
+        public abstract string Name { get; set; }
```

Comment:
No need to do Name abstract or virtual since you don't change logic at all in inheritors.
Logic for property is code that is located inside get and set block.